docs/data/sdks/node/index.md

```diff
@@ -4,7 +4,7 @@ description: The Amplitude Node.js SDK installation and quick start guide.
 icon: simple/nodedotjs
 ---
 
-![npm version](https://badge.fury.io/js/%40amplitude%2Fnode.svg)
+[![npm](https://img.shields.io/npm/v/@amplitude/node)](https://www.npmjs.com/package/@amplitude/node)
```

Comment:
This is what I got
```
[![npm version](https://badge.fury.io/js/@amplitude%2Fnode.svg)](https://badge.fury.io/js/@amplitude%2Fnode)
```

Reply:
1. the badge.fury.io has an issue in the doc. This comes from https://shields.io/. Please check this thread for the discussion. https://amplitude.slack.com/archives/C02T742UUMC/p1678302120754409
2. Justin pointed out, we should redirect the customer to the library, for example, redirect to npm for ts, and redirect to maven central to android. I noticed it works right in the readme.md doc in each GitHub. For example, https://github.com/amplitude/Amplitude-Kotlin/edit/main/README.md. I noticed it's not able to redirect in the staging environments. So I want to test it out on production.